src/sources/exec/mod.rs

```diff
@@ -210,13 +216,18 @@ impl ExecConfig {
             Some(config) => config.respawn_interval_secs,
         }
     }
+
+    fn log_namespace(&self) -> LogNamespace {
+        self.log_namespace.unwrap_or(false).into()
+    }
```

Comment:
nit but I'm not seeing a _huge_ advantage to this function? AFAICT it's only used once and the body is just a one-liner :shrug:

Reply:
I think it'll also default to `None` anyways and be fine?